tests/drivers/build_all/display/testcase.yaml

```diff
@@ -1,9 +1,10 @@
 common:
+  tags:
+    - drivers
+    - display
   build_only: true
-  min_ram: 32
-  tags: drivers display
+  platform_allow:
+    - native_posix
```

Comment:
Not sure we need native_posix.

Reply:
I copied it from the other build_all samples, seems like most of them still have it, I'd imagine Alberto will drop them all at some stage anyway.